BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Models/ResourceModel.cs

```diff
@@ -121,6 +126,9 @@ private AttributeModel GetManufacturer()
                     return new AttributeModel("device.manufacturer", "Apple");
                 case RuntimePlatform.Android:
                     return new AttributeModel("device.manufacturer", AndroidNative.GetManufacturer());
+                case RuntimePlatform.WindowsEditor:
+                case RuntimePlatform.WindowsPlayer:
+                    return new AttributeModel("device.manufacturer", "PC");
```

Comment:
Has this been agreed as the string to use?

Reply:
this is what we use for the notifier

Reply:
@gareththackeray – happy with this? It feels a lot more vague than other platforms, but consistency with the errors notifier is probably more important at this stage. Maybe a future enhancement?

Reply:
Let's leave it as "PC" so it's consistent. It's probably not even possible to get this information from the Unity API anyway. Looks like we send all sorts of different strings in for other platforms (e.g. Android manufacturers), so shouldn't need any pipeline changes.